db/migrate/20161103101916_create_applications.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :applications do |t|
       t.string :motivation
       t.references :user, index: true, foreign_key: true
-      t.references :workshop, index: true, foreign_key: true
+      t.references :event, index: true, foreign_key: true
```

Comment:
> Do we really need an additional migration for this?

I don't think so. I assume @Baschdl's comment was more of a general comment for renaming fields. This field however **is** already renamed, so changing the migration is not necessary and should be reverted.